themes/mms-cloud/page.html

```diff
@@ -42,7 +42,33 @@
 
 {%- block onboarding_modal -%}{%- endblock -%}
 
-{%- block alertbar -%}{%- endblock -%}
+{%- block alertbar -%}
+    {%- if theme_has_mdb_agent %}
+        <div class="alert alert-info">
+          <span class="alert-message">
+
+            <h4 class="alert-heading">Change is coming soon to the MongoDB Agents!</h4>
+            Monitoring, Backup and Automation are merging to become
+            the MongoDB Agent.
```

Comment:
Deferring to Melissa on copy. Don't merge without her signoff.

Reply:
Agreed.